client/src/App.js

```diff
@@ -6,6 +6,7 @@ import { BrowserRouter, Route, Switch, Link, NavLink } from "react-router-dom";
 import HomePage from "./pages/HomePage";
 import SingnIn from "./pages/SignIn";
 import SignUp from "./pages/SignUp";
+import { signup } from "./assets/signup.svg";
```

Comment:
The reason the SVG's aren't loading up because you are importing it dynamically! I tested it and it works fine if you remove the curly braces.
So it should look something like...
`import signup from "./assets/signup.svg";`

> **_Note: The rest of the code in the image tag remains the same!_**